lib/models/project.js

```diff
@@ -82,21 +83,26 @@ Project.prototype.setupNodeModulesPath = function(){
   debug('nodeModulesPath: %s', this.nodeModulesPath);
 };
 
-var NULL_PROJECT = new Project(process.cwd(), {});
+var processCwd = process.cwd();
+Project.createNullProject = function (ui, cli) {
+  var NULL_PROJECT = new Project(processCwd, {}, ui, cli);
 
-NULL_PROJECT.isEmberCLIProject = function() {
-  return false;
-};
+  NULL_PROJECT.isEmberCLIProject = function() {
```

Comment:
No, not really.
The indentation is because null projects are now created by the `Project.createNullProject` function.
Maybe checking out the code without the diff makes it easier to see: https://github.com/rodyhaddad/ember-cli/blob/rodyhaddad/cli-configurable/lib/models/project.js#L87-L105

The reason for the method is because Null Projects need a reference to the `cli` instance of the app.